hack/build/build-go.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -e
+
+source hack/build/common.sh
```

Comment:
so we assume user is always at repo root...

Reply:
The assumption here is that it's being executed via the Makefile, so the user has to be in the root dir.